pyemma/coordinates/tests/test_nystroem_tica.py

```diff
@@ -82,5 +90,79 @@ def test(self):
         assert np.all(np.abs(self.tica_obj.eigenvalues[:4] - t.eigenvalues[:4]) < 1e-3)
 
 
+class TestNystroemTICA_oASIS(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.X = np.ones((10000, 10))
+        cls.variable_columns = np.random.choice(10, 5, replace=False)
```

Comment:
Thanks! But does the test also pass if you make the choice out of np.random.choice(cls.X.shape[1], 5, replace=False)

Reply:
How do you mean? (cls.X.shape[1] == 10)